include/libopencm3/sam/4l/pm.h

```diff
@@ -181,4 +181,14 @@
 #define PM_PBDMASK_EIC_SHIFT			4
 #define PM_PBDMASK_PICOUART_SHIFT			5
 
+enum mck_src {
+	MCK_SRC_RCSYS = 0,
+	MCK_SRC_OSC0,
+	MCK_SRC_PLL,
+	MCK_SRC_DFLL,
+	MCK_SRC_RC80M,
+	MCK_SRC_RCFAST,
+	MCK_SRC_RC1M,
+};
```

Comment:
Is MCK part of the power manager unit? it might be, just be careful with namespaces